huggingface/pytorch/inference/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -153,13 +154,15 @@ RUN pip install --no-cache-dir -U \
     "awscli<2" \
     boto3 \
     pyopenssl \
-    "cryptography>=38.0.0" \
+    "cryptography>=41.0.6" \
     enum-compat==0.0.3 \
     "numpy>=1.22.2,<1.23" \
     opencv-python \
     packaging \
     "Pillow>=9.0.0" \
     "pyyaml>=5.4" \
+    cryptography \
```

Comment:
sanity-check: isn't this going to override the pinned version on line 157?

Reply:
I will remove line 164, since it's a less strict requirement that line 157 will cover.